httpx/models.py

```diff
@@ -918,18 +919,34 @@ def __repr__(self) -> str:
         Read and return the response content.
         """
         if not hasattr(self, "_content"):
-            self._content = b"".join([part async for part in self.stream()])
+            self._content = b"".join([part async for part in self.stream_bytes()])
         return self._content
 
-    async def stream(self) -> typing.AsyncIterator[bytes]:
+    @property
+    def stream(self):  # type: ignore
+        warnings.warn(
+            "Response.stream() is due to be deprecated. "
+            "Use Response.stream_bytes() instead."
+        )
```

Comment:
PendingDeprecationWarning?

```suggestion
        , PendingDeprecationWarning)
```